docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -163,3 +164,12 @@ In this case, try increasing the memory for the {kib} instance to 2GB.
 === ARM systems
 
 Chromium is not compatible with ARM RHEL/CentOS.
+
+[float]
+[[reporting-troubleshooting-maps-ems]]
+=== Unable to connect to Elastic Maps Service
+
+https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)] is a service that hosts
```

Comment:
FYI I recently added some short-cuts for these terms in https://github.com/elastic/docs/pull/2115, in case you want to use them:
```suggestion
https://www.elastic.co/elastic-maps-service[{ems} ({ems-init})] is a service that hosts
```